patch/kconfig-secure-boot-inclusions

```diff
@@ -11,7 +11,7 @@ CONFIG_MODULE_SIG_FORCE=y
 CONFIG_SYSTEM_TRUSTED_KEYS="debian/certs/debian-uefi-certs.pem"
 CONFIG_MODULE_SIG_HASH="sha512"
 CONFIG_MODULE_SIG_SHA512=y
-CONFIG_KEXEC_SIG_FORCE=y
```

Comment:
Why is this a problem for arm64 and not amd64?

Reply:
Due to the difference in the architecture and set of the drivers

Reply:
CONFIG_KEXEC_SIG_FORCE is not available for arm64. It is basically copy-paste that no one tested

Reply:
I see. Approved